group_vars/bm_wrk_win.yml

```diff
@@ -1,3 +1,2 @@
-ov_osdp: 'WindowsServer2016_SingleFrame_v1.0.2'           # Name of the OneView Server Profile Template to use for worker nodes
-ov_template: 'Windows Worker Node (single-frame) v1.0.2'  # Name of the OS Deployment Plan, we should probably be able to figure it out
+ov_template: 'Windows Worker Node v1.0.3'                 # Name of the OS Deployment Plan, we should probably be able to figure it out
```

Comment:
please check the linux code. It figures it out!

Reply:
Figures what out?  You said the linux code figures out the deployment plan, which is why I removed the ov_osdp variable.  The Linux code changed between specifying the ov_template variable in the hosts file (one per host) vs. the bm_wrk_*.yml vars file.  I prefer the vars file since all bare metal workers should be using the same template.